### PR TITLE
Adopt PowerShell syntax on Windows

### PR DESCRIPTION
**Pull request by richlander**

### CODE REVIEW

File: samples/complexapp/README.md [hunk 1]
```diff
@@ -112,22 +111,26 @@ The following instructions demonstrate this scenario in various configurations,
 ### Linux or macOS
 
 ```console
-$ docker build --pull --target test -t complexapp-test .
-$ docker run --rm -v $(pwd)/TestResults:/source/tests/TestResults complexapp-test
+docker build --pull --target test -t complexapp-test .
+docker run --rm -v $(pwd)/TestResults:/source/tests/TestResults complexapp-test
```
Comment (SteveL-MSFT) [inline samples/complexapp/README.md]:
```suggestion
docker run --rm -v ${pwd}/TestResults:/source/tests/TestResults complexapp-test
```

File: samples/build-in-sdk-container.md [hunk 2]
```diff
@@ -48,29 +48,52 @@ dotnetapp.dll
...
-01/14/2020  01:38 PM               154 dotnetapp.runtimeconfig.json
-               5 File(s)        177,957 bytes
-               2 Dir(s)  50,011,774,976 bytes free
+docker run --rm -v ${pwd}:c:\app -w c:\app mcr.microsoft.com/dotnet/core/sdk:3.1 dotnet publish -c Release -o out
+```
+
+You can see the built binaries with the following command:
+
+```console
+> dir out
```
Comment (MichaelSimons) [inline samples/build-in-sdk-container.md]:
```suggestion
PS C:\git\dotnet-docker\samples\dotnetapp> dir out
```

File: samples/build-in-sdk-container.md [hunk 3]
```diff
@@ -92,28 +115,29 @@ dotnetapp.deps.json		dotnetapp.runtimeconfig.json
...
-               5 File(s)        177,459 bytes
-               2 Dir(s)  106,439,225,344 bytes free
+mkdir C:\dotnetapp
+docker run --rm -v C:\dotnetapp:/out -v ${pwd}:/app -w /app mcr.microsoft.com/dotnet/core/sdk:3.1 dotnet publish -c release -o /out -r win-x64 --self-contained false
 ```
 
+You can see the built binaries with the following command:
+
+```console
+> dir C:\dotnetapp\
```
Comment (MichaelSimons) [inline samples/build-in-sdk-container.md]:
```suggestion
PS C:\git\dotnet-docker\samples\dotnetapp> dir C:\dotnetapp\
```

File: samples/complexapp/README.md [hunk 4]
```diff
@@ -66,20 +66,20 @@ ENTRYPOINT ["dotnet", "test", "--logger:trx"]
 
 The presence of the `test` stage costs very little and doesn't significantly change the behavior of the build if you don't specifically target it. By default, the test stage `ENTRYPOINT` will not be used if you build this Dockerfile
 
-The following example demonstrates targeting the `test` stage with the `--target` argument, and with logging enabled:
+The following example demonstrates targeting the `test` stage with the `--target` argument, and with logging enabled, using PowerShell:
 
 ```console
-C:\git\dotnet-docker\samples\complexapp> docker build --pull --target test -t complexapp-test .
+PS > docker build --pull --target test -t complexapp-test .
```
Comment (MichaelSimons) [inline samples/complexapp/README.md]:
```suggestion
PS C:\git\dotnet-docker\samples\complexapp> docker build --pull --target test -t complexapp-test .
```

File: samples/complexapp/README.md [hunk 5]
```diff
@@ -66,20 +66,20 @@ ENTRYPOINT ["dotnet", "test", "--logger:trx"]
...
 Sending build context to Docker daemon  12.81MB
 Step 1/15 : FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS build
 3.1: Pulling from dotnet/core/sdk
 Successfully built f98c5453be3d
 Successfully tagged complexapp-test:latest
 SECURITY WARNING: You are building a Docker image from Windows against a non-Windows Docker host. All files and directories added to build context will have '-rwxr-xr-x' permissions. It is recommended to double check and reset permissions for sensitive files and directories.
 
-C:\git\dotnet-docker\samples\complexapp>mkdir TestResults
+PS > mkdir TestResults
```
Comment (MichaelSimons) [inline samples/complexapp/README.md]:
```suggestion
PS C:\git\dotnet-docker\samples\complexapp> mkdir TestResults
```

File: samples/complexapp/README.md [hunk 6]
```diff
@@ -66,20 +66,20 @@ ENTRYPOINT ["dotnet", "test", "--logger:trx"]
...
 
-C:\git\dotnet-docker\samples\complexapp>docker run --rm -v %cd%/TestResults:/source/tests/TestResults complexapp-test
+PS > docker run --rm -v $pwd/TestResults:/source/tests/TestResults complexapp-test
```
Comment (MichaelSimons) [inline samples/complexapp/README.md]:
```suggestion
PS C:\git\dotnet-docker\samples\complexapp> docker run --rm -v $pwd/TestResults:/source/tests/TestResults complexapp-test
```

File: samples/complexapp/README.md [hunk 7]
```diff
@@ -94,15 +94,14 @@ Total tests: 2
      Passed: 2
  Total time: 1.8321 Seconds
 
-C:\git\dotnet-docker\samples\complexapp>dir TestResults
- Volume in drive C is Windows
- Volume Serial Number is 384B-0B6E
+PS > dir .\TestResults\
```
Comment (MichaelSimons) [inline samples/complexapp/README.md]:
```suggestion
PS C:\git\dotnet-docker\samples\complexapp> dir .\TestResults\
```

File: samples/complexapp/README.md [hunk 8]
```diff
@@ -112,22 +111,26 @@ The following instructions demonstrate this scenario in various configurations,
 ### Linux or macOS
 
 ```console
-$ docker build --pull --target test -t complexapp-test .
-$ docker run --rm -v $(pwd)/TestResults:/source/tests/TestResults complexapp-test
+docker build --pull --target test -t complexapp-test .
```
Comment (mthalman) [inline samples/complexapp/README.md]: There are a number of `docker build` and `docker run` commands that could benefit from showing the shell's current directory as Michael suggests.

**Comment by richlander** [comment]
I went back and forth on this topic. I erred on the side of simplicity vs potentially helpful noise. There is no perfect answer. I'll take these changes.

**Pull request merged**